packages/dx-react-scheduler-material-ui/src/templates/common/overlay-container.test.jsx

```diff
@@ -1,23 +1,23 @@
 import * as React from 'react';
 import { createShallow, getClasses } from '@material-ui/core/test-utils';
-import { Container } from './container';
+import { OverlayContainer } from './overlay-container';
 
-describe('EditRecurrenceMenu', () => {
+describe('Common', () => {
   const defaultProps = {
     ref: React.createRef(),
   };
```

Comment:
According to `propTypes` this component has not `ref` property. And any default props